configs/default/SPBE-SPEEDYBEEF7V2.config

```diff
@@ -0,0 +1,134 @@
+# Betaflight / STM32F7X2 (S7X2) 4.2.5 Nov 22 2020 / 18:39:53 (afdac08b3) MSP API: 1.43
+
+board_name SPEEDYBEEF7V2
+manufacturer_id SPBE
+
+# resources
+resource BEEPER 1 B02
+resource MOTOR 1 A08
+resource MOTOR 2 A09
+resource MOTOR 3 A10
+resource MOTOR 4 C09
+resource PPM 1 B07
+resource LED_STRIP 1 B00
+resource SERIAL_TX 1 B06
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 B07
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource I2C_SCL 1 B08
+resource I2C_SCL 2 B10
+resource I2C_SDA 1 B09
+resource I2C_SDA 2 B11
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource CAMERA_CONTROL 1 A00
+resource ADC_BATT 1 C02
+resource ADC_RSSI 1 C03
+resource ADC_CURR 1 C01
+resource PINIO 1 A15
+resource FLASH_CS 1 A14
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource USB_DETECT 1 C15
+
+# timer
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A09 AF1
+# pin A09: TIM1 CH2 (AF1)
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
+timer C09 AF2
+# pin C09: TIM3 CH4 (AF2)
+timer B00 AF3
+# pin B00: TIM8 CH2N (AF3)
+timer A00 AF2
+# pin A00: TIM5 CH1 (AF2)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A08 1
+# pin A08: DMA2 Stream 1 Channel 6
+dma pin A09 1
+# pin A09: DMA2 Stream 2 Channel 6
+dma pin A10 1
+# pin A10: DMA2 Stream 6 Channel 6
+dma pin C09 0
+# pin C09: DMA1 Stream 2 Channel 5
+dma pin B00 0
+# pin B00: DMA2 Stream 2 Channel 0
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature LED_STRIP
+feature OSD
+feature ESC_SENSOR
+
+# serial
+serial 0 64 115200 57600 0 115200
+serial 3 1024 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 2
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
```

Comment:
Unlike the other proposed custom defaults, this one will lead to the risk of unintended motor spin up if the fligth controller is used with certain ESCs. For this reason it is only acceptable if it is for an all-in-one design that does not allow the user to use non-original ESCs in an easy way.
Furthermore, if this is left at the default, the design of Betaflight configurator is such that the user is prompted to select an appropriate ESC protocol on connect, making this a good user experience, and not likely to result in support requests:

![image](https://user-images.githubusercontent.com/4742747/101479127-8e599a00-39b6-11eb-9a6a-b46a159d9181.png)

Reply:
Got it, We'll add a tip to tell the customer to choose the esc protocol manually.